### PR TITLE
Duplicate calls to configure should merge

### PR DESCRIPTION
**Pull request by ruudk**
Failing test for #5496

### CODE REVIEW

**Comment by samsonasik** [comment]
@ruudk I cherry-picked your commit at PR https://github.com/rectorphp/rector/pull/5503

File: src/DependencyInjection/CompilerPass/MergeImportedRectorConfigureCallValuesCompilerPass.php [hunk 1]
```diff
@@ -35,12 +35,48 @@ public function process(ContainerBuilder $containerBuilder): void
...
             return;
         }
 
-        $definition->removeMethodCall(self::CONFIGURE_METHOD_NAME);
+        // The following code only removes the first method call named `configure`,
+        // and leaves the rest. It's a bug, see https://github.com/symfony/symfony/pull/40167
+        // $definition->removeMethodCall(self::CONFIGURE_METHOD_NAME);
+        $definition->setMethodCalls(array_filter($definition->getMethodCalls(), function(array $call) {
+            return $call[0] !== self::CONFIGURE_METHOD_NAME;
+        }));
```
Comment (samsonasik) [inline src/DependencyInjection/CompilerPass/MergeImportedRectorConfigureCallValuesCompilerPass.php:53]:
```suggestion
        $methodCalls = array_filter($definition->getMethodCalls(), function(array $call) {
             return $call[0] !== self::CONFIGURE_METHOD_NAME;
         });
        $definition->setMethodCalls($methodCalls);
```

Comment (ruudk) [reply src/DependencyInjection/CompilerPass/MergeImportedRectorConfigureCallValuesCompilerPass.php:53]: It's not ready, more commits coming.

**Comment by ruudk** [comment]
Closing this, will create a new one

**Comment by ruudk** [comment]
Continued in https://github.com/rectorphp/rector/pull/5523

**Pull request closed without merging**